model/task/db.go

```diff
@@ -2619,11 +2634,20 @@ func getTasksByVersionPipeline(versionID string, opts GetTasksByVersionOptions)
 			},
 		})
 	}
-
-	if opts.UseSlowAnnotationsLookup {
-		pipeline = append(pipeline, AddAnnotationsSlowLookup...)
-	} else {
-		pipeline = append(pipeline, AddAnnotations...)
+	// TODO: DEVPROD-___ Remove this conditional and annotation lookup pipelines
+	// Pull one task, any task associated with this versionID. A task's CreateTime is
+	// derived from the version commit time or the patch creation time, allowing us to treat it
+	// as a proxy for the version creation time.
+	dbTask, err := FindOne(db.Query(bson.M{VersionKey: versionID}))
```

Comment:
Why don't we pass in the `create_time` from the version when calling this function so that we can save this lookup?

Reply:
That would be the ideal way of doing this but the full version document isn't always available in all of the usages of this function. Since this is a temporary addition, rather than adding version db queries in all of those places I just added one query here to be removed later.